core/storage/skill/gae_models.py

```diff
@@ -150,3 +150,30 @@ class SkillSummaryModel(base_models.BaseModel):
     # model was created).
     skill_model_created_on = ndb.DateTimeProperty(required=True, indexed=True)
     version = ndb.IntegerProperty(required=True)
+
+
+class SkillsMasteryModel(base_models.BaseModel):
+    """Mastery model of a user for Oppia Skill.
+
+    This model stores progress of each skill for a given user
+    and returns skill id.
+    """
+
+    # The user id of the currently logged in user.
+    user_id = ndb.StringProperty(required=True, indexed=True)
+    # The skill id for which degree of mastery is stored.
+    skill_id = ndb.StringProperty(required=True, indexed=True)
+    # The degree of mastery of the user in a skill.
+    degree_of_mastery = ndb.FloatProperty(required=True, indexed=True)
+
+    @classmethod
+    def get_user_mastery(cls, user_id):
+        """Returns skill summary for a particular user.
+
+        Args:
+            user_id: str. The user id corresponding to a user.
+
+        Returns:
+            str. fetched objects where user_id belongs to the entered user.
```

Comment:
The type information is wrong; the docstring should also be more specific than "fetched objects".

Please also add a backend test for this to ensure that it is working correctly.

Reply:
```
Returns:
            list. All skill summary models where user_id belongs to the
                entered user are returned as a list where each element
                is one skill summary model.
```
How does this look to you?

Reply:
I'm happy to do a full review once you've addressed all the comments.

But I'll also repeat a comment from elsewhere, which is to not use the term "summary" for stuff that is not a summary. See https://github.com/oppia/oppia/pull/4997#discussion_r192647889

Reply:
Done. And I've used the name skill mastery model